tests/cross-compile.bats

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bats
+
+load './test_helpers/bats-support/load'
+load './test_helpers/bats-assert/load'
+
+@test "clean out any cross compiled binaries" {
```

Comment:
@ocean 
It looks to me that these 3 tests should be in a single test:
1. We clean the binaries and assert that they do not exist
2. We compile them
3. We assert that they were compiled
4. We clean them again and assert that they do not exist

This approach would test the full end-to-end command lifecycle.

Currently, "check cross compiled binaries exist" (3rd test) relies on the result of work of the "cross compile binaries with make" (2nd test), which means that test assets are "bleeding" through tests, making tests not self-contained.

Reply:
Heh, I did have them in two but broke them up more so it was easier to see the tests running.

However, I agree it's better to have it one test and therefore self contained. I looked to see if BATS could output progressive updates to show parts of the test running, but it buffers all output to the end, so no.